src/Ast.ml

```diff
@@ -664,7 +671,7 @@ let location = function
   | Mod x -> x.pmod_loc
   | Sig x -> x.psig_loc
   | Str x -> x.pstr_loc
-  | Top -> Location.none
+  | Top | Tli _ -> Location.none
```

Comment:
There is a loc for `Item` and for `Directive`

Reply:
Done